doc/guides/arch/arc-support-status.rst

```diff
@@ -0,0 +1,84 @@
+|                                                                     | **EM**     | **HS3x/4x** | **EV** | **HS6x** |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Port status                                                         | upstreamed | upstreamed  | WIP    | WIP      |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| **Features**                                                                                                       |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Closely coupled memories (ICCM, DCCM) [#f1]_                        | Y          | Y           | TBD    | TBD      |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Execution with caches - Instruction/Data, L1/L2 caches              | Y          | Y           | Y      | Y        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Hardware-assisted unaligned memory access                           | Y [#f2]_   | Y           | TBD    | Y        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Regular interrupts with multiple priority levels, direct interrupts | Y          | Y           | TBD    | Y        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Fast interrupts, separate register banks for fast interrupts        | Y          | Y           | TBD    | N        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Hardware floating point unit (FPU)                                  | Y          | Y           | N      | N        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Symmetric multiprocessing (SMP) support, switch-based               | N/A        | Y           | TBD    | WIP      |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Hardware-assisted stack checking                                    | Y          | Y           | TBD    | N        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| Hardware-assisted atomic operations                                 | N/A        | Y           | TBD    | Y        |
++---------------------------------------------------------------------+------------+-------------+--------+----------+
+| DSP extension                                                       | N [#f3]_   | N [#f3]_    | TBD    | TBD      |
```

Comment:
Can you split this into 1) DSP ISA, and 2) DSP AGU/XY?
E.g. EM5D/7D are fully supported (accumulator save/restore), the 'N' in the table is confusing as it only applies to the AGU/XY single task restriction (maybe some lazy context switching like for FPU could be an option).